gatherers/censys.py

```diff
@@ -49,7 +49,7 @@
 wildcard_pattern = re.compile("^\*\.")
 redacted_pattern = re.compile("^(\?\.)+")
 
-def gather(suffix, options):
+def gather(suffix, options, extra={}):
```

Comment:
The ability to use the `url` gatherer under different names meant changing the method signature for each gatherer's `gather` function to include an optional `extra` dict, which might have information that doesn't appear directly in the command line flags. 

In this case, the `url` gatherer's `gather` method has to know, at call-time, which of the names it's operating under, so that it can pull the right CSV URL/path from the right command line flag.